tests/meson.build

```diff
@@ -84,6 +84,17 @@ if gtest_dep.found()
 
     test('unittest_sink', unittest_sink, timeout: 120, env: testenv)
 
+    # RUN unittest_tensorRegion
+    if tflite2_support_is_available
```

Comment:
This condition is not necessary because we do not use tf-lite tensor filter. We just use its results. So, Let's remove this condition.